src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+.main {
+  margin: 0;
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: 200px;
+  justify-content: center;
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
The media queries are well implemented for different breakpoints. However, make sure that the fixed column width of `200px` fits the design requirements. If the content requires more space, consider using a percentage or a more flexible unit like `fr` for the `grid-template-columns`.